docs/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ nav_order: 5
 
 ## main
 
+## 3.13.1
+
+* Look for preview files that end in `preview.rb` rather than `_preview.rb`, so they can be put in the sidecar directory alongside test files.
+
```

Comment:
```suggestion
* Look for preview files that end in `preview.rb` rather than `_preview.rb`, so they can be put in the sidecar directory alongside test files.

    *Seth Herr*

```